### PR TITLE
Feature/show chip value

### PR DESCRIPTION
**Pull request by corneliusgdahling**
Show chips based on values from data.js

### CODE REVIEW

File: web-app/src/App.svelte [hunk 1]
```diff
@@ -5,13 +5,15 @@
...
 	import data from './data';
 	import Fullscreen from './Fullscreen.svelte';
+	import Chips from './Chips.svelte';
 
 	const timeFromUser = data.timePerRound;
 </script>
 
 <style>
-	main {
+	#mainContainer {
```
Comment (corneliusgdahling) [inline web-app/src/App.svelte:14]:
```suggestion
	main {
```

File: web-app/src/invertHex.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export const invertHex = hex => {
+    return '#' + (Number(`0x1${hex.replace('#','')}`) ^ 0xFFFFFF).toString(16).substr(1).toUpperCase()
+  }
```
Comment (simjohan) [inline web-app/src/invertHex.js:3]: Why do we need this? (Just curious)

File: web-app/src/ChipIcon.svelte [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+				stroke={color}
+				fill={color} />
+			<circle cx="93.372002" cy="53.498001" r="8" id="circle83" fill={color} />
+			<circle cx="38.757999" cy="148.382" r="8" id="circle85" fill={color} />
+			<circle cx="93.623001" cy="243.123" r="8" id="circle87" fill={color} />
+			<circle cx="203.105" cy="242.97701" r="8.0010004" id="circle89" fill={color} />
+			<circle cx="257.71701" cy="148.091" r="8" id="circle91" fill={color} />
+			<circle cx="202.853" cy="53.351002" r="8" id="circle93" fill={color} />
+		</g>
+	</svg>
```
Comment (simjohan) [inline web-app/src/ChipIcon.svelte:85]: Why dont you use chip.svg?

**Review by simjohan — CHANGES_REQUESTED** [review]
I think we need to find a cleaner way to display chip values.
In my opinion the chip icons are way too large and we need to think about the colours.

**Comment by simjohan** [comment]
And whats up with inactive fullscreen?
Screenshot 2020-08-27 at 12 44 44

File: web-app/src/invertHex.js (hunk 2 above)
Comment (corneliusgdahling) [reply web-app/src/invertHex.js:3]: Should probably be changed to calculate contrast. I added it as a quick way to get a color for the chip value, to be automatically different to chip color (so it is legible).

**Review by espenmeidell — CHANGES_REQUESTED** [review]
Extra package lock? Don't think we need :/

File: web-app/src/invertHex.js [hunk 4]
```diff
@@ -0,0 +1,3 @@
+export const invertHex = hex => {
```
Comment (eddern) [inline web-app/src/invertHex.js:1]: Don't put this file in src/ root

**Review by eddern — CHANGES_REQUESTED** [review]
Real nice
- Delete either `web-app/src/images/chip.svg` or code in `web-app/src/ChipIcon.svelte`
- Move InvertHex logic somewhere else (maybe inside component folder)
Keep up the good work

**Review by eddern — APPROVED** [review]

**Pull request merged**